rules/prefer-t-throws.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const {visitIf} = require('enhance-visitors');
+const createAvaRule = require('../create-ava-rule');
+const util = require('../util');
+
+function hasAwaitExpression(nodes) {
```

Comment:
There must be a simpler way to achieve this.

Reply:
I don't know, but the function does a bit more than its name indicates: it looks for an AwaitExpression, which is not in a (arrow) function, and throws if it meets a return.

Reply:
At least add a code comments that succinctly explains what the function does.